src/chrome/content/rules/Auctiva.com.xml

```diff
@@ -1,15 +1,31 @@
 <!--
-	CDN buckets:
+	Invalid certificate:
+		betatest.auctiva.com
+		l.e.auctiva.com
+		eduimg.auctiva.com
+		livechat.auctiva.com (broken chain)
 
-		- img.auctiva.com.s3.amazonaws.com
+	No working URL known:
+		counters.auctiva.com
+		scimg.auctiva.com
```

Comment:
There is a list of working url on `https://scimg.auctiva.com/` itself.

Reply:
I still can't find any working URL :(. The content on https://scimg.auctiva.com/ doesn't work.

Reply:
`http://scimg.auctiva.com/imgsc/0/0/0/0/0/8/sc/110024537942.jpg`